docs/en/integrations/data-visualization/tableau/tableau-and-clickhouse.md

```diff
@@ -32,7 +32,7 @@ follow the setup guide bellow.
    of <a href="https://github.com/ClickHouse/clickhouse-java/releases/" target="_blank">ClickHouse JDBC driver</a>.
 
 :::note
-Make sure you download the **clickhouse-jdbc-x.x.x-shaded.jar** JAR file. Currently, we recommended using version `0.4.6`.
+Make sure you download the **clickhouse-jdbc-x.x.x-shaded-all.jar** JAR file. Currently, we recommended using versions `0.8.X`.
```

Comment:
```suggestion
Make sure you download the **clickhouse-jdbc-x.x.x-shaded-all.jar** JAR file.
```
Is there a good reason to have recommended version not be latest?

Reply:
The testing in half-manual and we did it with `0.8.1` only